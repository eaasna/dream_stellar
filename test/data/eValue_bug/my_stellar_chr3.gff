3	Stellar	eps-matches	1	173	95.4022	+	.	1;seq2Range=686,859;eValue=4.79405e-73;cigar=169M1I4M;mutations=49G,57C,71C,89C,97T,165A,170A,173G
2	Stellar	eps-matches	206	280	96.0526	+	.	1;seq2Range=1102,1176;eValue=8.65029e-30;cigar=3M1I2M1D69M;mutations=4T,17G
2	Stellar	eps-matches	209	281	95.9459	+	.	1;seq2Range=448,521;eValue=9.34656e-29;cigar=71M1I2M;mutations=44C,61G,72G
1	Stellar	eps-matches	489	560	95.8333	+	.	1;seq2Range=287,356;eValue=1.22629e-27;cigar=2D70M;mutations=13G
1	Stellar	eps-matches	351	421	95.7746	+	.	1;seq2Range=217,286;eValue=4.03092e-27;cigar=70M1D;mutations=6A,11T
1	Stellar	eps-matches	211	280	95.7142	+	.	1;seq2Range=1244,1313;eValue=1.325e-26;cigar=70M;mutations=11G,16T,62C
1	Stellar	eps-matches	491	560	95.8333	+	.	1;seq2Range=287,358;eValue=1.22629e-27;cigar=69M1I1M1I;mutations=13G,70C,72T
2	Stellar	eps-matches	351	420	95.7142	+	.	1;seq2Range=520,589;eValue=1.09117e-26;cigar=70M;mutations=22C,31T,61G
2	Stellar	eps-matches	358	411	96.3636	+	.	1;seq2Range=912,966;eValue=1.73628e-20;cigar=52M1I2M;mutations=26A,53T
1	Stellar	eps-matches	351	403	96.2264	+	.	1;seq2Range=1314,1366;eValue=2.27804e-19;cigar=53M;mutations=5A,22C
1	Stellar	eps-matches	215	265	96.0784	+	.	1;seq2Range=1643,1693;eValue=2.46139e-18;cigar=51M;mutations=8G,24T
1	Stellar	eps-matches	491	539	95.9183	+	.	1;seq2Range=1841,1889;eValue=2.65951e-17;cigar=49M;mutations=20C,36A
1	Stellar	eps-matches	96	140	95.5555	+	.	1;seq2Range=1199,1243;eValue=3.10487e-15;cigar=45M;mutations=2G,42T
1	Stellar	eps-matches	385	420	97.2222	+	.	1;seq2Range=1805,1840;eValue=3.91658e-12;cigar=36M;mutations=2A
3	Stellar	eps-matches	1	31	96.7741	+	.	1;seq2Range=590,620;eValue=5.38983e-10;cigar=31M;mutations=24C
2	Stellar	eps-matches	71	93	95.6521	+	.	1;seq2Range=152,173;eValue=1.68702e-05;cigar=20M1D2M;mutations=
2	Stellar	eps-matches	399	420	95.4545	+	.	1;seq2Range=664,685;eValue=5.54538e-05;cigar=22M;mutations=19C
2	Stellar	eps-matches	90	110	95.238	+	.	1;seq2Range=1056,1076;eValue=0.000182281;cigar=21M;mutations=12A
2	Stellar	eps-matches	102	121	95	+	.	1;seq2Range=1068,1087;eValue=0.000599173;cigar=20M;mutations=10A
2	Stellar	eps-matches	129	141	100	+	.	1;seq2Range=1095,1107;eValue=0.0699511;cigar=13M;mutations=
1	Stellar	eps-matches	270	280	100	+	.	1;seq2Range=206,216;eValue=0.917775;cigar=11M;mutations=
1	Stellar	eps-matches	72	81	100	+	.	1;seq2Range=1921,1930;eValue=3.0168;cigar=10M;mutations=
1	Stellar	eps-matches	551	560	100	+	.	1;seq2Range=1901,1910;eValue=3.0168;cigar=10M;mutations=
2	Stellar	eps-matches	422	431	100	+	.	1;seq2Range=539,548;eValue=2.48442;cigar=10M;mutations=
3	Stellar	eps-matches	34	43	100	+	.	1;seq2Range=623,632;eValue=1.08184;cigar=10M;mutations=
2	Stellar	eps-matches	368	377	100	-	.	1;seq2Range=1105,1114;eValue=2.48442;cigar=10M;mutations=
