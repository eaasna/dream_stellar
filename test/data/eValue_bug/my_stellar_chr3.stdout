I/O options:
  database file   : chr3.fasta
  query file      : query_e0.05.fasta
  alphabet        : dna5
  output file     : my_stellar_chr3.gff
  output format   : gff

User specified parameters:
  minimal match length             : 10
  maximal error rate (epsilon)     : 0.05 = (1/20)
  maximal x-drop                   : 5
  search forward strand            : yes
  search reverse complement        : yes

  verification strategy            : exact
  maximal number of matches        : 50
  duplicate removal every          : 500
  threads                          : 1

Calculated parameters:
  k-mer length : 10
  s^min        : 10
  threshold    : 1
  distance cut : 10
  delta        : 16
  overlap      : 0

Loaded 1 query sequence.
Loaded 1 database sequence.

All matches matches resulting from your search have an E-value of: 
        1.08184 or smaller  (match score = 1, error penalty = -2)

Constructing index...

Aligning all query sequences to database sequence...
  3
lengthAdjustment:
15
lengthAdjustment:
15
lengthAdjustment:
15
  3, complement

# Eps-matches     : 3
